src/rules/accessibility-label.js

```diff
@@ -34,31 +34,30 @@ module.exports = {
     schema: [schema]
   },
 
-  create: (context: ESLintContext) => ({
-    JSXOpeningElement: (node: JSXOpeningElement) => {
-      const accessible = getProp(node.attributes, 'accessible');
-      if (accessible) {
-        const labelPropVal = getAccessibilityLabel(node);
-        if (!labelPropVal) {
-          let childWithLabel;
-          // $FlowFixMe
-          if (node.parent) {
-            childWithLabel = findChild(node.parent, child => {
-              if (child.attributes) {
-                const childLabelValue = getAccessibilityLabel(child);
-                return !!childLabelValue;
-              }
-              return false;
-            });
-          }
-          if (!childWithLabel) {
-            context.report({
-              node,
-              message: errorMessage
-            });
-          }
-        }
-      }
-    }
-  })
+	create: (context: ESLintContext) => ({
+		JSXOpeningElement: (node: JSXOpeningElement) => {
+			const accessible = getProp(node.attributes, 'accessible');
+			if (!accessible || (accessible.value && accessible.value.expression.value === false)) return;
+			const labelPropVal = getAccessibilityLabel(node);
+			if (!labelPropVal) {
+				let childWithLabel;
+				if (node.parent) {
```

Comment:
Flow is failing here as it does not recognise `parent`, can we add a `$FlowFixMe` please? I think this was added in a recent commit but has maybe been overridden accidentally?

Reply:
Added